.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,18 @@
-## Testing
+## 📝 Summary
 
 <!--
```

Comment:
I'm not sure the summary title is needed? When you make a PR from a branch, it auto puts the commit message at the beginning. If there is a summary title, I'd have to copy and move it under the Summary title every time. I sort of feel the summary is already implied. But keep the instructions regarding the summary.